### DISCUSSION TITLE
Not working after updating to 24h2

### DISCUSSION THREAD

**Discussion by Jenry16** [rocksdanister/lively#2464] [Q&A]
Since I updated to Windows 11 24H2 the wallpaper stopped appearing, normally I have my desktop completely clean by right clicking and disabling show icons, but when I have this disabled the wallpaper stops appearing when I start the PC (when I have the option to see the icons on the desktop enabled the wallpapers do appear), when I disable seeing the icons on the desktop the wallpaper appears and I can disable the icons on the desktop again and it works normally, but it is annoying to have to do this every time I start the PC to have a clean desktop

**Comment by rocksdanister** [comment]
This is a Windows issue, as a workaround enable desktop icons:
right-click desktop -> View -> Show desktop icons.

Windows updates "KB5050009" and "KB5050094" have the fixes it seems, open Windows update settings -> Check for update.
https://support.microsoft.com/topic/january-14-2025-kb5050009-os-build-26100-2894-bdbfb097-ea20-487d-9171-718d15e26f1b

**Edit:**
Try the latest beta release:
https://github.com/rocksdanister/lively-beta/releases

**Reply by rocksdanister** [reply]
One workaround would be to leave the icons ON and write a script to adjust desktop icons opacity to 0.

Or just wait for the Windows update.

**Reply by danutzzzzz** [reply]
I have this issue but enabling desktop icons doesnt fix the issue

**Reply by exodus3155** [reply]
I clicked show desktop icons and it started working then i went back and clicked hide desktop icons and it stayed working till I reboot then I just do the same thing again. My pc never goes into standby or sleep but I supposed if it did I would have to repeat the process again. I love your app it's the best thank you Rocksdanister

**Reply by danutzzzzz** [reply]
> I clicked show desktop icons and it started working then i went back and clicked hide desktop icons and it stayed working till I reboot then I just do the same thing again. My pc never goes into standby or sleep but I supposed if it did I would have to repeat the process again. I love your app it's the best thank you Rocksdanister

thanks, that doesnt work for me, what version of windows you on? Windows [Version 10.0.26100.2454]

@rocksdanister you sure this is the same issue?

**Reply by rocksdanister** [reply]
I still think so (don't ping please.)

~Last I checked it was working for me in insider Version 24H2 (OS Build 26120.2122)~
Wrong, require icons enabled.. I'll check again.

**Reply by danutzzzzz** [reply]
Well it doesn't work at all for me, irrelevant of whether the ICON's are showing or hidden.

The worker keeps crashing as per my original post.

**Reply by Sn1perzOnYT** [reply]
> One workaround would be to leave the icons ON and write a script to adjust desktop icons opacity to 0.
> 
> Or just wait for the Windows update.

Really Weirdly This Worked Idk How But It Worked

**Reply by danutzzzzz** [reply]
ok i have Microsoft Windows [Version 10.0.26100.2454] on both my laptop and my PC.. Lively works fine on my Laptop but not my PC.. so my original post: https://github.com/rocksdanister/lively/issues/2407 is still valid.

**Reply by rocksdanister** [reply]
Windows update can still have different features (stage rollout) in same version.

The original error there just says WorkerW which is a Windows component getting destroyed, this is unusual and usually happens if something changes in Windows or some third party app is doing something.

**Reply by danutzzzzz** [reply]
These are both fresh installs from the same media.. how would i tell?

Just to advise, that interestingly i left the WorkerW issue to loop/crash for a few minutes and it suddenly started working. however its inconsistent with a reboot.

**Reply by Vogicator** [reply]
> Windows update "KB5050009" have the fix it seems, open Windows update settings -> Check for update. https://support.microsoft.com/topic/january-14-2025-kb5050009-os-build-26100-2894-bdbfb097-ea20-487d-9171-718d15e26f1b

Hi !
I wanted to install the Windows update because I had the same problem and the installer told me that the update was already installed.
I went to the performance settings and like some, I disabled and then re-enabled the "animate" boxes but the problem remained the same. While Lively was briefly displaying an image in a loop behind my window, I disabled "animate" and suddenly everything worked correctly!! 😊
You have to disable "animate" for Lively to work with the KB5050009 update. After... I haven't restarted my machine yet. 😅
![image](https://github.com/user-attachments/assets/82c27fd2-ab81-4559-b04c-90a69ea7ba28)

**Reply by Vogicator** [reply]
I confirm that after restarting Windows, Lively works perfectly!
I tried to check the 3 boxes and it works.
I restarted again to check.
So you have to uncheck the 3 boxes, restart Windows, then you can recheck the 3 boxes. I hope this will help those who have been struggling for several months.

**Reply by megapetera** [reply]
Thank you for your help, I used your method and it worked perfectly! many thanks!

**Reply by Tvoe-afk** [reply]
Helped me as well

**Comment by exodus3155** [comment]
Edition Windows 11 Pro
Version 24H2
Installed on ‎11/‎22/‎2024
OS build 26100.2454
Experience Windows Feature Experience Pack 1000.26100.36.0

**Comment by Hermione69** [comment]
Same mine also not working. check out the video i recorded - https://streamable.com/nif6gm
i tried uninstalling and reinstalling but still not working.

**Reply by XxUnknownSkillxX** [reply]
So for me it worked by changing my screen resolution from 2560x1440 to 1920x1080 and then back up again but thats only a temporary fix because once i restarted my pc it was bugged again

**Comment by edvaldo-filho** [comment]
Edição	Windows 11 Pro
Versão	24H2
Instalado em	‎02/‎01/‎2025
Compilação do SO	26100.2894
Experiência	Pacote de Experiência de Recursos do Windows 1000.26100.36.0

Same for me, i recorded the issue : https://youtu.be/7FMrVxSs1Xw

**Comment by danutzzzzz** [comment]
`Microsoft Windows [Version 10.0.26100.2605]` seems to be working for me.. anyone else?

**Comment by edvaldo-filho** [comment]
Seen to be resolved restarting explorer.exe, and if the desktop icons are hidden, they need be shown, then goes run normally and can go back w hidden icons after it (see the comments on the video report that was recorded: https://youtu.be/7FMrVxSs1Xw)

**Reply by danutzzzzz** [reply]
i dont need to do any of that since `Microsoft Windows [Version 10.0.26100.2605]`

**Comment by MechanicalEngineering2141** [comment]
Not working here either on 24H2. Can set videos as wallpaper, but anything with graphics/webGL doesn't work. 
I always have desktop icons showing by default too. Also tried after the KB5050009 update and still nothing.
Any ideas what could be the issue here? Or just have to wait for a windows update?

https://github.com/user-attachments/assets/c4d8c5c9-f644-471a-b7fb-e7cc4128d9a9

**Comment by rocksdanister** [comment]
Try the latest beta release:
https://github.com/rocksdanister/lively-beta/releases

**Reply by Wagnerah** [reply]
It works tks!
